src/sections/body/body.section.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react"
+import Sidebar from "../../components/sidebar/sidebar.component"
+import Table from "../../components/table/table.component"
+
+const Body = () => {
+  const [tableContent, setTableContent] = useState({
+    Mon: {
+      Breakfast: {content:'', rounded:"tl", b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:"tr", b:true, e:false},
+    },
+    Tue: {
+      Breakfast: {content:'', rounded:null, b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:null, b:true, e:false},
+    },
+    Wed: {
+      Breakfast: {content:'', rounded:null, b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:null, b:true, e:false},
+    },
+    Thur: {
+      Breakfast: {content:'', rounded:null, b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:null, b:true, e:false},
+    },
+    Fri: {
+      Breakfast: {content:'', rounded:null, b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:null, b:true, e:false},
+    },
+    Sat: {
+      Breakfast: {content:'', rounded:null, b:true, e:true},
+      Lunch: {content:'', rounded:null, b:true, e:true},
+      Dinner: {content:'', rounded:null, b:true, e:false},
+    },
+    Sun: {
+      Breakfast: {content:'', rounded:"bl", b:false, e:true},
+      Lunch: {content:'', rounded:null, b:false, e:true},
+      Dinner: {content:'', rounded:"br", b:false, e:false},
+    }
+  })
+
+  const [showAltForm, toggleAlt] = useState({ showAlt:false, meal:'', day:'', content:'' })
+
+  const toggleShowAltForm = (meal, day, content, bool) => {
+    toggleAlt({
```

Comment:
the function says `toogleShowAltForm` but you're not just toggling the form, you're also updating the meal, day and content. so i'd suggest you should either 
- rename it so it's clearer what this function does/can do
- have a separate state for toggling and another for the state of the form itself